src/sentry/apidocs/examples/replay_examples.py

```diff
@@ -77,19 +77,19 @@ class ReplayExamples:
                     {
```

Comment:
could we also keep an example where there isn't a component name? could just be another elem in the `data` array